src/__test__/add_delete_edit_check.test.js

```diff
@@ -0,0 +1,79 @@
+/**
+@jest-environment jsdom
+*/
+import store from '../modules/store.js';
+import displayTodo from '../modules/function.js';
+import formClear from '../modules/cleartask.js';
+
+document.body.innerHTML = '<input type="text" name="task" class="text-field full" placeholder="Add to your list..." required> <ul class="task-list"> </ul> <a class="clear-btn" href="">Clear all completed</a>';
+
+describe('Store', () => {
+  test('Should add element when its called', () => {
+    store('Hello');
+    displayTodo();
+    const list = document.querySelectorAll('textarea');
+    expect(list).toHaveLength(1);
+  });
+});
+
+describe('Delete task', () => {
+  test('Should remove added task', () => {
+    const deleteBtn = document.querySelector('.delete');
+    deleteBtn.click();
+    const listElement = document.querySelectorAll('textarea');
+    expect(listElement).toHaveLength(0);
+  });
+});
```

Comment:
- [ ] Well! Your function is working well when I have a single to-do and want to delete it. However, when I have multiple todos, visually I can delete any todo item. In your test function, I can only delete the first one. Please add a condition to check if I can delete other list items when I have multiple to-do items.  Keep up the good work and ensure your functions handle different scenarios appropriately! 👍🧪🚀

Reply:
completed!!